zppy/e3sm_diags.py

```diff
@@ -197,6 +203,14 @@ def e3sm_diags(config, scriptDir, existing_bundles, job_ids_file):  # noqa: C901
                         ts_sub = c["ts_subsection"]
                     else:
                         ts_sub = c["sub"]
+
+                    if (
+                        "ts_daily_subsection" in c.keys()
+                        and c["ts_daily_subsection"] != ""
+                    ):
+                        ts_sub = c["ts_daily_subsection"]
```

Comment:
@chengzhuzhang Did you mean for this to be `ts_sub`. I think using `ts_daily_sub` would be better, no? I think there's a naming collision with `ts_sub` above.